### DISCUSSION TITLE
Why is alpha-nvim disabled?

### DISCUSSION THREAD

**Discussion by sini-kettu** [NvChad/NvChad#1261] [General]
In the [plugins init file](../blob/main/lua/plugins/init.lua#L158), `alpha-nvim` is disabled, and simply enabling it allows it to install and work properly if enabled in `chadrc.lua`. Why is it disabled? I like having the dashboard when I open Nvim and I'm curious if this was a mistake or not. 

It looks like it was changed two months ago in [this commit](../blame/d327582dc033759e7a5c4a0046d88a168e1da97a/lua/plugins/init.lua#L158), but again, I fail to see how this influences the re-implementation of the config file.

When searching through discussions, I see in #615 that startup time was "a lil slow" so I suppose that could be the reason for disabling it.

**Comment by siduck** [comment]
this isnt a mistake, many dont like it but its a nice UI feature and enabling it is still an easy task so those who want it can just enable it

**Reply by sini-kettu** [reply]
Is there a better way to enable it (e.g. in the custom folder) than editing `init.lua` in the plugins folder?

**Reply by siduck** [reply]
@little512 enable it in your plugin section ,https://nvchad.github.io/config/Custom%20config#enable-dashboard

**Reply by sini-kettu** [reply]
Perfect, thanks.

**Reply by julian-avar** [reply]
Moved here: https://nvchad.github.io/config/plugins#enable-dashboard
